apps/pocketbase/migrations/1679449550_updated_maps.go

```diff
@@ -0,0 +1,55 @@
+package migrations
+
+import (
+	"encoding/json"
+
+	"github.com/pocketbase/dbx"
+	"github.com/pocketbase/pocketbase/daos"
+	m "github.com/pocketbase/pocketbase/migrations"
+	"github.com/pocketbase/pocketbase/models/schema"
+)
+
+func init() {
+	m.Register(func(db dbx.Builder) error {
+		dao := daos.New(db);
+
+		collection, err := dao.FindCollectionByNameOrId("tgcm0cd7l7jj0ze")
+		if err != nil {
+			return err
+		}
+
+		// add
+		new_image := &schema.SchemaField{}
+		json.Unmarshal([]byte(`{
+			"system": false,
+			"id": "q3odadwo",
+			"name": "image",
+			"type": "file",
+			"required": false,
+			"unique": false,
+			"options": {
+				"maxSelect": 1,
+				"maxSize": 5242880,
+				"mimeTypes": [
+					"image/webp"
+				],
+				"thumbs": []
+			}
+		}`), new_image)
+		collection.Schema.AddField(new_image)
+
+		return dao.SaveCollection(collection)
+	}, func(db dbx.Builder) error {
+		dao := daos.New(db);
+
+		collection, err := dao.FindCollectionByNameOrId("tgcm0cd7l7jj0ze")
```

Comment:
Should `FindCollectionByNameOrId("tgcm0cd7l7jj0ze")` and `collection.Schema.RemoveField("q3odadwo")` be hardcoded? They could change with each instance or yapms.

Reply:
From my understanding, the ID should be constant.

I didn't write these migration files, pocketbase automatically creates them when you modify the database.

You can see the "collections_snapshot.go" file it will generate a table called maps with the "tgcm0cd7l7jj0ze" id. So anyone who runs pocketbase will have the same ID.

Then it will run through all the migration files to make the correct modifications to the maps database.

But these files aren't written by hand, its all handled by pocketbase.

![image](https://user-images.githubusercontent.com/13600696/227726645-588fa43e-90b7-44ff-ac7d-28ccff57dc32.png)

Reply:
This function enables automigrations.

![image](https://user-images.githubusercontent.com/13600696/227726821-183b2629-4783-411e-ab2a-458b7a188464.png)

Reply:
![aaaa](https://user-images.githubusercontent.com/13600696/227727003-4b33e940-a73c-46af-94dc-075795c375a3.gif)